package.json

```diff
@@ -10,11 +10,11 @@
   ],
   "scripts": {
     "build": "npm run clean && npm run build-version && npm run build-lib && npm run build-types && npm run build-webpack",
-    "build-lib": "babel src -d lib --source-maps --extensions '.ts,.js'",
```

Comment:
I suspect none of the changes in this file is necessary? Please revert the changes if that's the correct

Reply:
I'm a bit confused why this shows up in this way here.  it still shows in my local copy. I have changed this however in the other pull request for windows where I change it to \" instead of single quotes.

Reply:
I suspect Prettier did a run through the file, and changed format.

Reply:
Awww... sorry I didn't think of that.